consensus/replay_test.go

```diff
@@ -1271,9 +1275,12 @@ func TestHandshakeUpdatesValidators(t *testing.T) {
 			t.Error(err)
 		}
 	})
-	if err := handshaker.Handshake(proxyApp); err != nil {
+	version, err := handshaker.Handshake(proxyApp)
+	if err != nil {
 		t.Fatalf("Error on abci handshake: %v", err)
 	}
+	require.Equal(t, version, customVersion)
```

Comment:
[nit] the order of arguments to require.Equal makes a difference if the assertion fails. The second argument should be the expected value and the third argument should be the actual value.

This is very minor but it makes it less confusing to read the message when an assertion fails.

```suggestion
	require.Equal(t, customVersion, version)
```

<img width="946" alt="Screenshot 2023-07-13 at 10 18 26 AM" src="https://github.com/celestiaorg/celestia-core/assets/3699047/1ba88816-67fe-4501-bd71-989d2e38f3e6">